src/pages/signIn/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslation } from 'react-i18next';
+import Button from 'src/modules/General/components/Button';
+import Icon from 'src/modules/General/components/Icon';
+import { Input } from 'src/modules/General/components/Input';
+import { Link } from 'src/modules/General/components/Link';
+
+import css from './index.module.scss';
+import { useSignIn } from './useSignIn';
+
+export const SignIn = () => {
+  const { register, errors, handleSubmit } = useSignIn();
+  const { t } = useTranslation();
+
+  return (
+    <div className={css['container']}>
+      <div className={`w-full lg:min-w-[600px] ${css['formSection']}`}>
+        <img src={'/images/logo/logo.svg'} alt="SHIN_LOGO" width="79px" height="32px" />
```

Comment:
put our logos into `assets` and import it directly not with `URL` because:
- build-time optimizations and bundling
- prevent to render when relative path is wrong and easier access